name-as/configs/mips-linux-gnu-as.toml

```diff
@@ -1,6 +1,6 @@
 config_name = "Assemble with mips-linux-gnu-as"
 as_cmd = [
-    "mips-linux-gnu-as {INPUT_AS} -o a.o -mips32r6",
+    "mips-linux-gnu-as {INPUT_AS} -o a.o",
```

Comment:
what did this do and why was it removed?

Reply:
It specified which MIPS version to assemble, that version is incorrect. Had it there to test the differences. Default is -mips1 which is what MARS seems to be assembling (just comparing outputs)